### PR TITLE
Misc queue from 2018 (sorry, lost sources)

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -9286,6 +9286,7 @@ moreso->more, more so,
 morever->moreover
 morgage->mortgage
 morisette->morissette
+mormalized->normalized
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add the s based variant please and the singulars (s and z).

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -9983,6 +9986,7 @@ orded->ordered
 orderd->ordered
 orgamise->organise
 organim->organism
+organisaion->organisation
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add the z variant please.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -10218,6 +10223,7 @@ packhage->package
 packhages->packages
 packtes->packets
 pading->padding
+paermissions->permissions
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the singular please.

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -10947,6 +10954,7 @@ prefernces->preferences
 preferr->prefer
 preferrable->preferable
 preferrably->preferably
+preferrence->preference
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the plural please.

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -11285,6 +11293,7 @@ propertise->properties
 propertly->properly, property,
 propertys->properties
 propertyst->properties
+propertyvlaue->propertyvalue
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Again seems overly specific.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: emoved

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: There's no mormalised or mormalises in here yet.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Could we have the plural too in s and z?

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -12230,6 +12247,7 @@ pakge->package
 pakvage->package
 palatte->palette
 palce->place, palace,
+palces->places
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest pales

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Pull request merged**